### ISSUE TITLE
[test-runner] Report more details when failing coverage thresholds

### ISSUE THREAD

**Issue by web-padawan** [modernweb-dev/web#688]
From https://polymer.slack.com/archives/CE6D9DN05/p1601887288223100

I have the following config
``` 
threshold: {
  statements: 95,
  branches: 59,
  functions: 90,
  lines: 95
}
```
And locally it gives me this

```
Code coverage: 85.58 %
View full coverage report at coverage/lcov-report/index.html
```

But in CI it’s different and I have no idea what fails

```
Code coverage: 85.49 %
View full coverage report at coverage/lcov-report/index.html

Finished running tests in 44.9s, failed to meet coverage threshold.
```

It would be nice to have an option to show e.g. `Coverage for branches failed` to make it easier to investigate.

**Comment by LarsDenBakker** [comment]
When coverage does not pass, we should print the separate coverage % per threshold. 

Those changes can be made here: https://github.com/modernweb-dev/web/blob/master/packages/test-runner-cli/src/reporter/getCodeCoverage.ts

**Comment by web-padawan** [comment]
Fixed by #772
